data/migrations/20191219124316_article-table.js

```diff
@@ -0,0 +1,17 @@
+
+exports.up = function(knex) {
+  return knex.schema.createTable("articles", table => {
+      table.integer("id").notNullable().unique();
```

Comment:
Should the id of the article not be a primary key? 
```suggestion
      table.increments();
```

Reply:
We'll be getting the articleId from the frontend using uuid.  I will change the datatype to uuid.